src/playground-ide.ts

```diff
@@ -254,6 +254,12 @@ export class PlaygroundIde extends LitElement {
   @property()
   pragmas: 'on' | 'off' | 'off-visible' = 'on';
 
+  /**
+   * The HTML file used in the preview.
+   */
+  @property()
```

Comment:
We've been following the convention that attributes are snake-case, so we should add (also in the other spot):

```suggestion
  @property({attribute: 'html-file')
```